homeassistant/components/opentherm_gw/services.yaml

```diff
@@ -7,6 +7,22 @@ reset_gateway:
       description: The gateway_id of the OpenTherm Gateway.
       example: "opentherm_gateway"
 
+set_central_heating_ovrd:
+  description: >
+    Set the central heating override option on the gateway.
+    When overriding the control setpoint (via a set_control_setpoint service call with a value other than 0), the gateway automatically enables the central heating override to start heating.
+    This service can then be used to control the central heating override status.
+    To return control of the central heating to the thermostat, call the set_control_setpoint service with temperature value 0.
+    You will only need this if you are writing your own software thermostat.
+  fields:
+    gateway_id:
+      description: The gateway_id of the OpenTherm Gateway.
+      example: "opentherm_gateway"
+    ch_override:
+      description: >
+        The desired boolean value for the central heating override.
+      example: "on"
```

Comment:
```suggestion
      example: true
```

Reply:
Any motivation for this suggestion? Both values achieve exactly the same thing: https://github.com/home-assistant/core/blob/157f972d722d4772e9b20f37829e10968fcb6ca8/homeassistant/helpers/config_validation.py#L132
Also, on/off is widely used in HA sensors, templates, etc. and feels more natural for non-programmers in my opinion.

Reply:
Any motivation for this suggestion? Both values achieve exactly the same thing: https://github.com/home-assistant/core/blob/157f972d722d4772e9b20f37829e10968fcb6ca8/homeassistant/helpers/config_validation.py#L132
Also, on/off is widely used in HA sensors, templates, etc. and feels more natural in my opinion.

(Comment re-added after accidentally deleting it before)

Reply:
Looks like we have it both ways though-out the code base.  `true` makes more sense to me, but I can see your point of view that I'm not the user.